tests/integration-tests/tests/storage/test_fsx_lustre/test_existing_fsx/s3_test_file

```diff
@@ -0,0 +1 @@
+Downloaded by FSx Lustre
```

Comment:
What is the purpose of the file?

Reply:
This file is to test `import_path` and `export_path`, which are used for FSx to link it to a durable data repository in Amazon S3. So users are able to Sync the data between FSx and S3 bucket. For more information, here is the doc:
https://docs.aws.amazon.com/fsx/latest/LustreGuide/create-fs-linked-data-repo.html